vagrant_variables.yml.atomic

```diff
@@ -15,20 +15,11 @@ memory: 1024
 
 disks: "[ '/dev/sdb', '/dev/sdc' ]"
 
-eth: 'enp0s3'
-# VAGRANT BOX
-# Fedora: https://download.fedoraproject.org/pub/fedora/linux/releases/22/Cloud/x86_64/Images/Fedora-Cloud-Base-Vagrant-22-20150521.x86_64.vagrant-virtualbox.box
-# Ubuntu: ubuntu/trusty64
-# CentOS: chef/centos-7.0
-# Debian: deb/jessie-amd64 - be careful the storage controller is named 'SATA Controller'
-# For more boxes have a look at:
-#   - https://atlas.hashicorp.com/boxes/search?utf8=✓&sort=&provider=virtualbox&q=
-#   - https://download.gluster.org/pub/gluster/purpleidea/vagrant/
-
+eth: 'enp0s8'
```

Comment:
so the interface changed from `enp0s3` to `enp0s8`?
Just checking for a typo :)